workos/directory_sync.py

```diff
@@ -131,6 +131,23 @@ def get_user(self, user):
 
         return WorkOSDirectoryUser.construct_from_response(response).to_dict()
 
+    def get_primary_email(self, user):
+        """Gets primary email for a single provisioned Directory User.
+
+        Args:
+            user (str): Directory User unique identifier.
+
+        Returns:
+            str: User's primary email response from WorkOS.
+        """
+        response = self.request_helper.request(
```

Comment:
We shouldn't be making an API request here.

The `get_primary_email` method on the `WorkOSDirectoryUser` is an instance method that the developer can call on the directory users returned from the other methods (like `get_user`).